### PR TITLE
Added Unit tests for ONSwipeTouchListener #87

### PR DESCRIPTION
**Pull request by Earthii**
These are the unit tests for the swipe feature

### CODE REVIEW

**Comment by Earthii** [comment]
I like this test. It first fetches the listener object from the surface view and once you have the listener object, we can invoke the listener methods to test if it had changed the appropriate state of the camera. 

Good job on this one

File: app/src/test/java/Test_OnSwipeTouchListener.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    val mockParameters = Mockito.mock(Camera.Parameters::class.java)
+    private lateinit var testPreview : Preview
+    var myListener : OnSwipeTouchListener? = null
+
+    @Before
+    fun setUp() {
+        activity = Robolectric.setupActivity(MainActivity::class.java)
+        surfaceView = activity!!.findViewById<SurfaceView>(R.id.camera_view)
+        testPreview = Preview(activity, surfaceView, activity)
+        testPreview.testingPurposeOnly_setDummyCamera(mockCamera, mockParameters)
```
Comment (Earthii) [inline app/src/test/java/Test_OnSwipeTouchListener.kt]: You don't need this line because we aren't doing anything camera related

**Comment by Earthii** [comment]
I resolved my review since @younesstawil is unavailable due to him being at work

File: app/src/test/java/Test_OnSwipeTouchListener.kt [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    var myListener : OnSwipeTouchListener? = null
+
+    @Before
+    fun setUp() {
+        activity = Robolectric.setupActivity(MainActivity::class.java)
+        surfaceView = activity!!.findViewById<SurfaceView>(R.id.camera_view)
+        testPreview = Preview(activity, surfaceView, activity)
+
+        // get the mListenerInfo in the View class
+        var listenerInfoField : Field
```
Comment (n2chao) [inline app/src/test/java/Test_OnSwipeTouchListener.kt]: Is there a reason why you declare and initialize listenerInfoField on two different lines?
Same question for myLiObject.

**Comment by Earthii** [comment]
I addressed all the review comments

**Pull request merged**